rpc/core/src/error.rs

```diff
@@ -47,6 +47,9 @@ pub enum RpcError {
     #[error("Rejected transaction {0}: {1}")]
     RejectedTransaction(RpcTransactionId, String),
 
+    #[error("Transaction {0} has an input with an empty signature_script at index {1}.")]
+    EmptySignatureScript(RpcTransactionId, usize),
```

Comment:
it can be extended to find all indexes that include empty signatures, WDYT?

Reply:
Yeah, I had this thought in the back of my head, but somehow didn't bring that up, perhaps in order to keep this KISS.  But yeah, @KaffinPX it might make sense to enumerate all inputs and return a concatenated string containing all indexes. U can use `.filter_map(|(index,input)| input.signature_script.is_empty().then(index)).collect::<Vec<_>>();` and react if the resulting vec is not empty... (or along those lines).  Then do `indexes.iter().map(String::from).join(",")` to get a single string that can be included in the error.